src/server/typingsInstaller/typingsInstaller.ts

```diff
@@ -192,6 +185,7 @@ namespace ts.server.typingsInstaller {
                 if (this.log.isEnabled()) {
                     this.log.writeLine(`Loaded content of '${packageJson}': ${JSON.stringify(npmConfig)}`);
                 }
+                const typingsCacheIsForCurrentVersion = this.areVersionsEqualMajorMinor(npmConfig.typeScriptVersion, ts.version);

```

Comment:
Variable only used inside `if` block, can be moved inside.
Actually, `for (const key in undefined) {}` won't crash, so the `if` should be unnecessary?